main.tf

```diff
@@ -87,9 +87,14 @@ data "aws_iam_policy_document" "bucket_policy" {
   }
 }
 
+module "aggregated_policy" {
+  source           = "git::https://github.com/cloudposse/terraform-aws-iam-policy-document-aggregator.git?ref=tags/0.1.2"
```

Comment:
count  = "${var.enabled == "true" ? 1 : 0}"

let's add `count` to this resource as well.
Because if `var.enabled` is set to `false`, all resources w/o a count=0 will still be created by terraform.

Reply:
This is a bit tidier; done.  No need for the state to have that intermediate resource if not needed.